demisto_sdk/commands/prepare_content/preparers/marketplace_incident_to_alert_playbooks_prepare.py

```diff
@@ -0,0 +1,127 @@
+            data, False
+        )
+
+        return data
+
+    @staticmethod
+    def prepare_descriptions_and_names(data: dict, replace_incident_to_alert: bool):
+
+        for k, v in data.get("tasks", {}).items():
+            if description := v.get("task", {}).get("description", ""):
+                if replace_incident_to_alert:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.replace_incident_to_alert(
+                        data, "description", description, k
+                    )
+                else:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.remove_wrapper_from_incident(
+                        data, "description", description, k
+                    )
+            if name := v.get("task", {}).get("name", ""):
+                if replace_incident_to_alert:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.replace_incident_to_alert(
+                        data, "name", name, k
+                    )
+                else:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.remove_wrapper_from_incident(
+                        data, "name", name, k
+                    )
+
+        # The external description
+        if description := data.get("description"):
+            if replace_incident_to_alert:
+                MarketplaceIncidentToAlertPlaybooksPreparer.replace_incident_to_alert(
+                    data, "description", description
+                )
+            else:
+                MarketplaceIncidentToAlertPlaybooksPreparer.remove_wrapper_from_incident(
+                    data, "description", description
+                )
+
+        # The external name
+        if name := data.get("name"):
+            if replace_incident_to_alert:
+                MarketplaceIncidentToAlertPlaybooksPreparer.replace_incident_to_alert(
+                    data, "name", name
+                )
+            else:
+                MarketplaceIncidentToAlertPlaybooksPreparer.remove_wrapper_from_incident(
+                    data, "name", name
+                )
+
+    @staticmethod
+    def replace_incident_to_alert(
+        data: dict,
+        name_or_description_field: str,
+        name_or_description_field_content: str,
+        key: str = None,
+    ):
+        mapping = {r"(?<!<-)incident(?!->)": "alert",
+                   r"(?<!<-)Incident(?!->)": "Alert",
+                   r"(?<!<-)incidents(?!->)": "alerts",
+                   r"(?<!<-)Incidents(?!->)": "Alerts"}
+
+        new_content = name_or_description_field_content
+        for pattern in mapping:
+            new_content = re.sub(
+                pattern, mapping[pattern], new_content
+            )
+
+        if key:
+            data["tasks"][key]["task"][name_or_description_field] = new_content
+        else:
+            data[name_or_description_field] = new_content
+
+    @staticmethod
+    def remove_wrapper_from_incident(
```

Comment:
this one is very similar to the one above - can we merge them? Maybe keep the mappings as constant at top of the file and use the appropriate one for each usecase?